engine/userguide/eng-image/dockerfile_best-practices.md

```diff
@@ -52,12 +55,27 @@ should avoid installing extra or unnecessary packages just because they
 might be “nice to have.” For example, you don’t need to include a text editor
 in a database image.
 
-### Run only one process per container
-
-In almost all cases, you should only run a single process in a single
-container. Decoupling applications into multiple containers makes it much
-easier to scale horizontally and reuse containers. If that service depends on
-another service, make use of [container linking](../../userguide/networking/default_network/dockerlinks.md).
+### Each container should have only one concern
+
+Decoupling applications into multiple containers makes it much easier to scale
+horizontally and reuse containers.  For instance, in a web application stack you
+might want three separate containers, each with its own unique image, to manage
+the web application, database, and an in-memory cache in a decoupled manner.
+
+You may have heard that there should be "one process per container".  While this
+mantra has good intentions, it is not strictly true that there should be only
+one operating system process per container.  In addition to the fact that
+containers can now be [spawned with an init
+process](https://docs.docker.com/engine/reference/run/#/specifying-an-init-process),
+there are some programs such as Apache and nginx which might spawn additional
```

Comment:
Some programs such as Apache web server and Nginx might spawn....

Maybe use an example that's not a web server so people don't assume this only applies to web servers!